### DISCUSSION TITLE
Push ke Github

### DISCUSSION THREAD

**Discussion by daryraa** [teamalgoritma/community#182] [Q&A] [answered]
mengenai capstone project telebot...Permisi saya ingin bertanya apakah push file itu sama seperti kita mengupload files via github?(upload files), dan untuk file bernama .gitignore itu dapat darimana ya? disaya tidak ada

**Comment by triarm** [comment, marked as answer]
Hi @daryraa ,
Iya betul, push file ke github dapat dilakukan dengan upload files. Untuk file .gitignore dapat diabaikan saja.

Demikian yang dapat kami sampaikan. Semoga menjawab.

**Reply by daryraa** [reply]
oke baik mas terima kasih
